Batch8CucumberSeleniumAPI/src/test/resources/features/ui/Search.feature

```diff
@@ -23,4 +23,14 @@ Feature: Search functionality
 		When I search for product as "Dell"
 		And I click on any product in the Search Result
 		Then I am able to see product description and detail in new tab
+		
+		@TC0023
+		Scenario: Verify after clicking on cart displayed in the header, the page is navigated to Shopping cart 
+		Given I have browser opened and url is navigated
+		When I search for product as "Dell"
+		And I click on add to cart 
+		Then product displayed in the header
+		Then The Page Is Navigated To Shopping Cart 
```

Comment:
Merge this step wither with "Then product displayed in the header" step or change step - "Then The Page Is Navigated To Shopping Cart " to And

Reply:
ok